scmrepo/git/__init__.py

```diff
@@ -363,6 +363,8 @@ def add_commit(
     validate_git_remote = partialmethod(_backend_func, "validate_git_remote")
     check_ref_format = partialmethod(_backend_func, "check_ref_format")
 
+    get_tree_obj = partialmethod(_backend_func, "get_tree_obj")
```

Comment:
This is only used in tests now, but in the future we should make `get_fs` not default to `GitFileSystem`'s default behaviour that uses pygit2.